lib/usd/ui/layerEditor/saveLayersDialog.cpp

```diff
@@ -189,20 +186,6 @@ void SaveLayerPathRow::onOpenBrowser()
 
 void SaveLayerPathRow::onTextChanged(const QString& text) { _absolutePath = text; }
 
-void SaveLayerPathRow::onRelativeButtonChecked(bool checked)
-{
-    if (checked) {
-        QDir dir(_initialStartFolder);
-
-        QString relativePath = dir.relativeFilePath(_absolutePath);
-        _pathEdit->setText(relativePath);
-        _pathEdit->setEnabled(false);
-    } else {
-        _pathEdit->setEnabled(true);
-        _pathEdit->setText(_absolutePath);
-    }
-}
```

Comment:
Just to confirm - this looks like a method that was never actually connected to any button. So it was dead code you removed?

Reply:
Yes, probably copied originally from LayerPathRow::onRelativeButtonChecked and never cleaned up.